velox/exec/HashProbe.cpp

```diff
@@ -2037,6 +2037,13 @@ void HashProbe::clearBuffers() {
   output_.reset();
   nonSpillInputIndicesBuffer_.reset();
   spillInputIndicesBuffers_.clear();
+  filterResult_.clear();
+  filterResult_.resize(1);
```

Comment:
Why do we need to resize to 1 after clear()? Is it for pre-allocation for next round probing? Can we add some comments here?

Reply:
Just keep it the same before the reclaim.